spectractor/simulation/atmosphere.py

```diff
@@ -614,7 +642,7 @@ def simulate(self, ozone, pwv, aerosols, angstrom_exponent=None):
             ...     title=a.title, label=a.label)
             >>> if parameters.DISPLAY: plt.show()
         """
-        if angstrom_exponent is not None and angstrom_exponent > 0:
+        if angstrom_exponent is not None and angstrom_exponent < 0:
```

Comment:
I'm not sure I understand the change from `>` to `<`. The error message makes it sound like anything that is not `None` should cause a problem, but this check is doing something different than the words suggest.